{{ cookiecutter.resource_name }}/api.py

```diff
@@ -15,20 +15,54 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 
-
 """API for manipulating {{ cookiecutter.resource_name }}."""
 
-from __future__ import absolute_import, print_function
+from functools import partial
+
+from .models import {{ cookiecutter.class_name }}Identifier
+from ..fetchers import id_fetcher
+from ..minters import id_minter
+from ..providers import Provider
+from ..api import IlsRecord, IlsRecordsSearch
+
+# provider
+{{ cookiecutter.class_name }}Provider = type(
+    '{{ cookiecutter.class_name }}Provider',
+    (Provider,),
+    dict(identifier={{ cookiecutter.class_name }}Identifier, pid_type='pers')
```

Comment:
`pid_type='pers')` This you need to update pid type  here

Reply:
Done